src/web3/BaseClient.ts

```diff
@@ -198,7 +190,7 @@ export class BaseClient {
 				const maxGasEncoded = Buffer.from(varintEncode((data as ICallData).maxGas));
 
 				// coins to send
-				const coinsEncoded = Buffer.from(varintEncode((data as ICallData).coins));
+				const coinsEncoded = Buffer.from(varintEncode(toMAS((data as ICallData).coins)));
```

Comment:
This breaks a lot of things especially in the deployer we need to be careful when making the update maybe it should be a major.